sci-libs/hdf5/hdf5-1.14.6-r1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=fortran
+
+inherit cmake fortran-2 flag-o-matic toolchain-funcs
+
+MY_PV=${PV/_p/-}
+MY_P=${PN}-${MY_PV}
+MAJOR_P=${PN}-$(ver_cut 1-2)
+
+DESCRIPTION="General purpose library and file format for storing scientific data"
+HOMEPAGE="https://github.com/HDFGroup/hdf5/"
+SRC_URI="https://github.com/HDFGroup/hdf5/releases/download/${PN}_${MY_PV/-/.}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="NCSA-HDF"
+SLOT="0/310-cmake"
```

Comment:
Hopefully the `-cmake` suffix can be dropped if it's fully compatible? Or does it break ABI?